installer/templates/new/assets/javascripts/widget.js

```diff
@@ -23,6 +23,17 @@ class Widget extends React.Component {
           setTimeout((() => window.location.reload()), 5 * 60 * 1000)
         }
       });
+
+      this._events.addEventListener('_kitto', (event) => {
```

Comment:
@Zorbash this works... sort of. The dashboard reloads as expected, but the reload event gets sent when the stream is opened, which means that right now the dashboard will just continually refresh. I'm not sure if there's a way to send the event and then expire it from the cache or not.